res/css/views/rooms/_EventTile.scss

```diff
@@ -106,6 +106,12 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
         }
     }
 
+    .mx_RoomView_searchResultsPanel & {
+        &.mx_EventTile_contextual {
```

Comment:
`mx_EventTile_contextual` is used for search results:

https://github.com/matrix-org/matrix-react-sdk/blob/37d8cfbc505bfba688b4101843cbb70a8c3de952/src/components/views/rooms/EventTile.tsx#L136-L138